product_brand/views/product_brand_view.xml

```diff
@@ -65,15 +65,15 @@
             </form>
         </field>
     </record>
-    <record id="view_product_brand_tree" model="ir.ui.view">
-        <field name="name">product.brand.tree</field>
+    <record id="view_product_brand_list" model="ir.ui.view">
```

Comment:
It is usually a bad idea to rename views XMLids and names, it may cause a lot of side effects.
As far as I know, Odoo has kept all "tree" refs in their view namings.
I think we should do the same.